src/components/DataCollector/index.js

```diff
@@ -13,6 +13,8 @@ governing permissions and limitations under the License.
 import createRequest from "../../core/createRequest";
 import createEvent from "../../core/createEvent";
 
+const VIEW_START_EVENT = "viewstart";
```

Comment:
We will have to decide on a naming convention here, so `viewstart` might become.

Reply:
What do you think about going camelCase with these event type values (e.g., `viewStart`)? I think I'd prefer it, probably because more complicated values will be easier to read.

Reply:
I agree.